File: cakk-api/src/main/java/com/cakk/api/dto/response/like/HeartCakeImageListResponse.java
package com.cakk.api.dto.response.like;

import java.util.List;

import lombok.Builder;

import com.cakk.domain.mysql.dto.param.like.HeartCakeImageResponseParam;

@Builder
public record HeartCakeImageListResponse(

	List<HeartCakeImageResponseParam> cakeImages,
	Long lastCakeHeartId,
	int size
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/like/HeartCakeImageListResponse.kt
package com.cakk.core.dto.response.like

import com.cakk.infrastructure.persistence.param.like.HeartCakeImageResponseParam

data class HeartCakeImageListResponse(
    val cakeImages: List<com.cakk.infrastructure.persistence.param.like.HeartCakeImageResponseParam>,
    val lastCakeHeartId: Long?,
    val size: Int
)
